### ISSUE TITLE
Update the ANSI SQL Compatibility page

### ISSUE THREAD

**Issue by rfraposa** [ClickHouse/clickhouse-docs#836]
This page here: https://clickhouse.com/docs/en/sql-reference/ansi/

It's tricky because it involves a lot of manual checking...but we should try our best to keep it up to date.

**Comment by gingerwizard** [comment]
Assigning to @melvynator @qoega as this is really abig lift and needs to be owned by PMs.

**Comment by qoega** [comment]
I agree, but we can't handle it without additional hands. I shared in DM in my opinion it should be covered by Melvyn's new hire or another one. To be done properly.

**Comment by Blargian** [comment]
Page no longer exists, closing
